src/passkey_child/passkey_child_assert.c

```diff
@@ -219,6 +219,67 @@ get_assert_auth_data_signature(TALLOC_CTX *mem_ctx, fido_assert_t *assert,
     return ret;
 }
 
+errno_t
+set_assert_auth_data_signature(const struct passkey_data *data,
+                               fido_assert_t *_assert)
+{
+    TALLOC_CTX *tmp_ctx = NULL;
+    const unsigned char *auth_data = NULL;
+    const unsigned char *signature = NULL;
+    size_t auth_data_len;
+    size_t signature_len;
+    errno_t ret;
+
+    tmp_ctx = talloc_new(NULL);
+    if (tmp_ctx == NULL) {
+        ret = ENOMEM;
+        goto done;
+    }
+
+    ret = fido_assert_set_count(_assert, 1);
```

Comment:
_assert denotes output variable, but it is used as input

Reply:
Same as in https://github.com/SSSD/sssd/pull/6493#discussion_r1090557415